alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -17,37 +22,80 @@ module Cardano.Ledger.Alonzo.Tx
     Tx (Tx, body, wits, isValidating, auxiliaryData),
     ScriptPurpose (..),
     Indexable (..),
+    --  Figure 5
```

Comment:
this is awesome lining up these exports with the figures from the spec!

Reply:
I have done an even better job, adding new figures.